utils/.config.template.toml

```diff
@@ -21,6 +21,8 @@ theme = { optional = false, default = "dark", example = "light", options = ["dar
 times_to_run = { optional = false, default = 1, example = 2, explanation = "Used if you want to run multiple times. Set to an int e.g. 4 or 29 or 1", type = "int", nmin = 1, oob_error = "It's very hard to run something less than once." }
 opacity = { optional = false, default = 0.9, example = 0.8, explanation = "Sets the opacity of the comments when overlayed over the background", type = "float", nmin = 0, nmax = 1, oob_error = "The opacity HAS to be between 0 and 1", input_error = "The opacity HAS to be a decimal number between 0 and 1" }
 transition = { optional = true, default = 0.2, example = 0.2, explanation = "Sets the transition time (in seconds) between the comments. Set to 0 if you want to disable it.", type = "float", nmin = 0, nmax = 2, oob_error = "The transition HAS to be between 0 and 2", input_error = "The opacity HAS to be a decimal number between 0 and 2" }
+sub_overlay = { optional = true, type = "bool", default = false, example = false, options = [true, false], explanation = "This will add a subscribe animation overlay to the end of the video" }
+sub_overlay_name = { optional = true, default = "youtube", options = ["youtube", "tiktok", "instagram"], explanation = "Choose which overlay you would like"}
```

Comment:
Could you possibly have it make multiple videos i.e. Make one with the Instagram overlay and make one with a YouTube overlay

Reply:
So you mean that it can also be random?

Reply:
No.

Reply:
For anybody needing additional context
![Screenshot_20220831-032902.png](https://user-images.githubusercontent.com/66544866/187619329-43d68459-0b0e-415b-b08e-2b0dca5fd9b7.png)

Reply:
If this is implemented, the videos should maybe save to a TikTok, a YouTube and a Instagram folder.

Reply:
so a good option would be to  render the video then before extracting the video we could check which overlay is specified and then possibly move to custom dir i.e TikTok/yt/insta